conu/apidefs/container.py

```diff
@@ -64,6 +67,15 @@ def http_request(self, path="/", method="GET", host=None, port=None, json=False,
         )
         return self.http_session.request(method, url, json=json, data=data)
 
+    @contextmanager
+    def http_client(self, host=None, port=None):
```

Comment:
2 things:
 * [ ] documentation (with an example)
 * [ ] at least one integration test (feel free to reuse some existing code)

Reply:
Done.